src/main/java/com/group4/vms/repository/TimesheetRepository.java

```diff
@@ -1,35 +1,63 @@
-package com.group4.vms.repository;
 
+package com.group4.vms.repository;
 
 import com.group4.vms.model.Entry;
+
+
 import org.bson.types.ObjectId;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.data.mongodb.core.query.Update;
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
 
 @Repository
 public class TimesheetRepository {
+
     @Autowired
     private MongoTemplate mongoTemplate;
 
-    public List<Entry> getAllEntries(){
-        return null;
-
+    public List<Entry> getAllEntries() {
+        return mongoTemplate.findAll(Entry.class);
     }
-    public List<Entry> getEntries(ObjectId[] entries){
-        return null;
-    }
-    public Entry getEntry(ObjectId id){
-        return null;
+
+    public List<Entry> getEntries(ObjectId[] entryIds) {
+        Query query = new Query(Criteria.where("_id").in((Object[]) entryIds));
```

Comment:
This query looks for a key called "_id" in an array of `ObjectIds`. You don't specify an `$is` operator. Additionally, Keys live and die in Mongo. The array is not a Hashmap. There are no key-value pairs in an array.